README.md

```diff
@@ -239,10 +242,20 @@ solc.loadRemoteVersion('latest', function(err, solcSnapshot) {
     // An error was encountered, display and quit
   } else {
     // NOTE: Use `solcSnapshot` here with the same interface `solc` has
+    // For example:
+    const output = solcSnapshot.compile(/* ... */)
   }
 });
 ```
 
+The version **must** be in the long format string.
+Thus, if you would like to use version `v0.8.17` you need to provide it, including the committed version of the release.
+You can extract the long version string for each version from the [public available release list](https://binaries.soliditylang.org/bin/list.json).
```

Comment:
I wanted to say that we should rather point people towards the new `emscripten-wasm32/` and `emscripten-asmjs/` dirs but I realized that it would be inconsistent because we're still using and mentioning `bin/` everywhere. So I guess it's out of scope of this PR. FYI I created an issue for it: #671.